furusiya/config.json

```diff
@@ -44,7 +44,7 @@
     },
     "enemies": {
         "arrowDropsOnKill": 3,
-        "randomlyWalkOutOfSight": false,
+        "randomlyWalkOutOfSight": true,

```

Comment:
This makes me think "randomly walk (until you are) out of sight," can you rename it to `randomlyWalkWhenNotSeen`?